src/platform-includes/sourcemaps/overview/javascript.remix.mdx

```diff
@@ -1,21 +1,71 @@
-Integrate your Remix project's source maps with Sentry using these steps:
+## Upload Source Maps for a Svelte Project
 
-### 1: Generate Source Maps
+Sentry uses [releases](/product/releases/) to match the correct source maps to your events.
+This page is a guide on how to create releases and upload source maps to Sentry when bundling your Remix app.
 
-Remix can be configured to output source maps using Remix CLI. Learn more about [how to generate source maps with Remix](./generating/).
+### 1. Generate Source Maps
 
-### 2: Provide Source Maps to Sentry
+To generate source maps with your Remix project, you need to call `remix build` with the `--sourcemap` option.
 
-Source maps can be uploaded to Sentry by creating a release. Learn more about [how to upload source maps](./uploading/).
+Please refer to the [Remix CLI docs](https://remix.run/docs/en/v1/other-api/dev#remix-cli) for more information.
 
+### 2. Create a Release and Upload Source Maps
 
-### 3: Remove Remix Source Maps 
+The Sentry Remix SDK provides an automatic script to create a release and upload source maps after you built your project.
+Under the hood, it uses the Sentry CLI.
+
+This script requires some configuration that can either be done through a `.sentryclirc` file in the root of your project or through environment variables:
+
+<Note>
+
+Visit the [auth token user settings page](https://sentry.io/settings/account/api/auth-tokens/) to find your auth tokens.
+
+</Note>
+
+```ìni {filename:.sentryclirc}
+[auth]
+token=your-auth-token
+
+[defaults]
+org=___ORG_SLUG___
+project=___PROJECT_SLUG___
+```
+
+```bash
+export SENTRY_AUTH_TOKEN=your-auth-token
+export SENTRY_ORG=___ORG_SLUG___
+export SENTRY_PROJECT=___PROJECT_SLUG___
+```
+
+Next, run the upload script with the following command:
+
+```yarn
+yarn sentry-upload-sourcemaps
+
+# For usage details run
+yarn sentry-upload-sourcemaps --help
+```
+
+```npm
+node ./node_modules/@sentry/remix/scripts/sentry-upload-sourcemaps.js
```

Comment:
Is there a reason we're not using `npm run` here? ([Docs](https://docs.npmjs.com/cli/v9/commands/npm-run-script))

Reply:
I don't think you can run binaries with npm run

Reply:
Ah, right, okay. Node it is, then!